nhais-adaptor/adaptor/incoming/operation_definition_adaptor.py

```diff
@@ -1,5 +1,5 @@
 from typing import List, Tuple
-
+import json

```

Comment:
Look like this import (and now the one below it) are not needed.

Reply:
👍 done